Objects/typeobject.c

```diff
@@ -4887,11 +4888,12 @@ object_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
 
         PyErr_Format(PyExc_TypeError,
                      "Can't instantiate abstract class %s "
-                     "without an implementation for abstract method%s %U",
+                     "without an implementation for abstract method%s '%U'",
                      type->tp_name,
                      method_count > 1 ? "s" : "",
                      joined);
         Py_DECREF(joined);
+        Py_DECREF(comma_w_quotes_sep);
```

Comment:
You also need to DECREF this in the two `return NULL` branches above.

Come to think of it, the second of those already leaks `joined`. It needs a `Py_DECREF(joined)`.

Reply:
Aah, yep! Fixed in [6ae052e](https://github.com/python/cpython/pull/97971/commits/6ae052ef771eb9e72f7cbb90b58a2bb1d10ae3f5).